### ISSUE TITLE
Add TA Attribute column to each simple parcel layer

### ISSUE THREAD

**Issue by palmerj** [linz/linz-lds-bde-schema#14]
Moved from https://github.com/linz/linz_bde_uploader/issues/17

Required access and use of offical StatsNZ TA boundaries](http://www.stats.govt.nz/browse_for_stats/Maps_and_geography/Geographic-areas/digital-boundary-files.aspx). One option is to use https://github.com/linz/download_ta_bdys and create_table_polygon_grid to make this possible

**Comment by strk** [comment]
Was this ticket deprecated ? We just dropped TA boundaries dependency from BDE processor (#197)

**Comment by palmerj** [comment]
No it's still a potential feature for users. Should ask @iharrison or @billgeo to comment. The can decide to keep it open or close it. The main benefit is users can download data for a specific TA by a simple attribute query in services such as WFS without having to use a slower and more complex geometry filter. 

Yes it would require a new way to get the TA boundaries and derive the attribute quickly each day.

**Comment by iharrison** [comment]
This is something LDS users have always asked for. I can't remember where our thinking got to on it but keep this open for now and we can close/update this once the team has discussed it.

**Comment by palmerj** [comment]
Cool. This feature might also be linked to a KxE improvement to allow file exports to include a attribute filter condition (either from custom input of a pre-processed enumerated list)

**Comment by palmerj** [comment]
Need to keep this open. @JBLINZ should also be aware of this long standing feature request. Of high value I think

**Comment by JBLINZ** [comment]
Ok. Makes sense. Seems like it should be easily achievable.

**Comment by palmerj** [comment]
The only issue is storage. A new column causes a write of the whole table forcing changesets for every row.

**Comment by strk** [comment]
Is this change still desired ?

**Comment by iharrison** [comment]
Yes I think so. I will raise it with the team to see if it's worth making a plan to do it this year.
